native/explorer/src/series.rs

```diff
@@ -1118,7 +1118,8 @@ pub fn s_not(s1: ExSeries) -> Result<ExSeries, ExplorerError> {
 
 #[rustler::nif(schedule = "DirtyCpu")]
 pub fn s_contains(s1: ExSeries, pattern: &str) -> Result<ExSeries, ExplorerError> {
-    Ok(ExSeries::new(s1.utf8()?.contains(pattern)?.into()))
+    // TODO: maybe have "strict" as an option.
```

Comment:
The docs don't say much :/

But for what I could understand, it is related to how the Regex is interpreted: https://pola-rs.github.io/polars/src/polars_ops/chunked_array/strings/namespace.rs.html#136

If it's "strict", then the regex must be valid. In case it's not strict and an invalid regex is used, it is going to search for "None" - nils.

Reply:
@philss for us `contain` is never a regex, so we need to make sure our `pattern` is not being interpreted as a regex.